### DISCUSSION TITLE
Reaction Identifiers

### DISCUSSION THREAD

**Discussion by Flash1593** [SysBioChalmers/Human-GEM#397] [General]
This might be a silly question but is there a way to identify or convert the reaction & metabolite IDs in the Human-GEM so the output in MATLAB comes as actual reaction & metabolite names rather than just Human-GEM identifier?

**Comment by JonathanRob** [comment]
@Flash1593 not a silly question at all! The `metNames` and `rxnNames` fields in the model contain names of metabolites and reactions, respectively, that correspond to the IDs. However, the `rxnNames` are unfortunately missing for many reactions, so it is not a complete/perfect solution.

There are also annotation files in the `model/` directory (`metabolites.tsv` and `reactions.tsv`) that provide additional identifiers for the metabolites and reactions. You can load them into Matlab using the `importTsvFile` function, or add the fields directly to the model using the `annotateGEM` function.
